SUS-941: replace skin preview images

mixth-sense opened the pull request:
JIRA case: https://wikia-inc.atlassian.net/browse/SUS-941
Code review: @sqreek

Commit:
refresh skin preview images

macbre commented:
Let's care about the front-end performance and bandwidth usage :)

```
$ sudo apt-get install pngcrush

$ pngcrush -brute aliencrate_preview.png  aliencrate_preview_out.png

   Best pngcrush method        =   6 (ws 15 fm 5 zl 9 zs 0) =     8097
      for aliencrate_preview_out.png
     (6.30% critical chunk reduction)
     (7.23% filesize reduction)
```

macbre commented on extensions/wikia/ThemeDesigner/templates/ThemeDesigner_ThemeTab.php:
```diff
@@ -7,12 +7,12 @@
 			<li data-theme="<?= $k ?>">
 				<label><?= ucfirst($k) ?></label>
-				<img src="<?= $wg->StylePath ?>/oasis/images/themes/<?= $k ?>_preview.jpg">
+				<img src="<?= $wg->StylePath ?>/oasis/images/themes/<?= $k ?>_preview.png">
```
Set `height` and `width` attributes.

sqreek commented:
except macbre comment, LGTM

Commit:
SUS-941 specify width and height to improve frontend performance

macbre commented:
As discussed offline, let's try `pngquant` that will generate 8-bit PNG with differing applied.

Commit:
pngquant images

Before:

8878	aliencrate_preview.png
618	babygirl_preview.png
3313	bluesteel_preview.png
725	carbon_preview.png
5915	creamsicle_preview.png
769	jade_preview.png
953	oasis_preview.png
14482	opulence_preview.png
10905	plated_preview.png
8264	police_preview.png
13062	rockgarden_preview.png

After:

5027	aliencrate_preview.png
406	babygirl_preview.png
3072	bluesteel_preview.png
503	carbon_preview.png
3654	creamsicle_preview.png
553	jade_preview.png
728	oasis_preview.png
7167	opulence_preview.png
6758	plated_preview.png
4106	police_preview.png
7159	rockgarden_preview.png

Commit:
replace new assets
